swtbot_test/src/iotLTStestsuites/ObjectGUICheckConstraints.java

```diff
@@ -72,13 +74,42 @@ public void tc_00_ChangeRTOSLocation() throws Exception{
 	
 	@Test
 	public void tc_01_CreateIoTLTSProject() throws Exception {
-		PGUtility.createProject(RTOSType.FREERTOSIOTLTS, RTOSVersion.IoTLTS_202210_1_0_0, RTOSApplication.IOT_LTS_ETHER_PUBSUB,
+		PGUtility.createProject(RTOSType.FREERTOSIOTLTS, RTOSVersion.IoTLTS_202210_1_0_0, RTOSApplication.KERNEL_BARE,
 				Constants.CCRX_TOOLCHAIN, TargetBoard.BOARD_CK_RX65N);
 	}
 	
 	@Test
 	public void tc_02_MustNotBeANumber() throws Exception {
-		Utility.MustNotBeANumber(projectModelSpecific, true);
```

Comment:
E: same as the above comment.
Q: Why not update this utility instead of removing it and write  new code?

Reply:
No, it is used in other RTOS project, but for IoT LTS case, this utility is not correct anymore, because there is a default task in newest IoT LTS package now. Therefore, we are no longer using this utility here
We are not updating the Utility because this Utility also use for other RTOS, and IoT LTS is different now. It is more convenient to keep this utility for others, and rewrite the code for this utility for IoT LTS specifically

Reply:
same as the above comment. move the new code to a utility for future maintenance

Reply:
I have fixed this in my latest patchset